src/components/Thanks/SingleVersion/BadgeMilestone.vue

```diff
@@ -134,7 +145,13 @@ const avatars = computed(() => props.loans.slice(0, 3));
 
 const showAvatars = computed(() => props.isOptedIn && avatars.value.length && !props.loanCommentModuleShown);
 
-const funFact = computed(() => displayedBadgeData.value.contentfulData?.shareFact ?? '');
+const badgeLevelName = computed(() => {
+	const levelData = getTierBadgeDataByLevel(displayedBadgeData.value, displayedBadgeData.value.level);
+	return levelData.tierName;
+});
+
+const funFact = computed(() => displayedBadgeData.value.contentfulData?.shareFact
+	|| 'Making a difference starts here. See your impact and achievements.');
```

Comment:
I guess this fun fact is in contentful...

Reply:
Do we need a fallback? What is the use case?

Reply:
If we don't need the fallback, we might as well remove it for now.

Reply:
Yeah, different copy on prod also...

<img width="953" alt="Screenshot 2025-02-14 at 11 53 18 a m" src="https://github.com/user-attachments/assets/31a1f42f-7158-4049-a6ee-c4658da59788" />